{
 "bad": false,
 "new_version": "1.4.4",
 "new_version_attempts": {
  "1.1.3": 1,
  "1.1.4": 1,
  "1.1.5": 2,
  "1.2": 2,
  "1.2.1": 1,
  "1.2.2": 1,
  "1.2.3": 1,
  "1.2.4": 1,
  "1.3": 1,
  "1.3.1": 1,
  "1.3.2": 1,
  "1.3.3": 0,
  "1.3.4": 3,
  "1.3.5": 1,
  "1.3.6": 1,
  "1.3.7": 1,
  "1.3.8": 1,
  "1.3.9": 1,
  "1.4.0": 5,
  "1.4.1": 31,
  "1.4.2": 3,
  "1.4.3": 5,
  "1.4.4": 5
 },
 "new_version_errors": {
  "1.4.0": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/9192307124\">bot CI job</a>): main: Traceback (most recent call last):\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 567, in _run_migrator_on_feedstock_branch\n    migrator_uid, pr_json = run(\n                            ^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 214, in run\n    migration_run_data = run_migration(\n                         ^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/migration_runner.py\", line 35, in run_migration\n    return run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/migration_runner.py\", line 132, in run_migration_local\n    migrator.run_pre_piggyback_migrations(recipe_dir, feedstock_ctx.attrs, **kwargs)\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/migrators/core.py\", line 418, in run_pre_piggyback_migrations\n    mini_migrator.migrate(recipe_dir, attrs, **kwargs)\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/migrators/conda_forge_yaml_cleanup.py\", line 42, in migrate\n    cfg = yaml.load(fp.read())\n          ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/main.py\", line 451, in load\n    return constructor.get_single_data()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 116, in get_single_data\n    return self.construct_document(node)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 125, in construct_document\n    for _dummy in generator:\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 1476, in construct_yaml_map\n    self.construct_mapping(node, data, deep=True)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 1366, in construct_mapping\n    if self.check_mapping_key(node, key_node, maptyp, key, value):\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 278, in check_mapping_key\n    raise DuplicateKeyError(*args)\nruamel.yaml.constructor.DuplicateKeyError: while constructing a mapping\n  in \"<unicode string>\", line 1, column 1:\n    build_platform: {osx_arm64: osx_64}\n    ^ (line: 1)\nfound duplicate key \"build_platform\" with value \"{'linux_aarch64': 'linux_64', 'linux_ppc64le': 'linux_64'}\" (original value: \"{'osx_arm64': 'osx_64'}\")\n  in \"<unicode string>\", line 3, column 1:\n    build_platform:\n    ^ (line: 3)\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\n",
  "1.4.1": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/9729117591\">bot CI job</a>): main:\nError running migrate-feedstock in container - error DuplicateKeyError raised:\n'while constructing a mapping',   in \"<unicode string>\", line 1, column 1:\n    build_platform: {osx_arm64: osx_64}\n    ^ (line: 1), 'found duplicate key \"build_platform\" with value \"{'linux_aarch64': 'linux_64', 'linux_ppc64le': 'linux_64'}\" (original value: \"{'osx_arm64': 'osx_64'}\")',   in \"<unicode string>\", line 3, column 1:\n    build_platform:\n    ^ (line: 3), '\n                    To suppress this check see:\n                        http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n                    ', '                    Duplicate keys will become an error in future releases, and are errors\n                    by default when using the new API.\n                    '",
  "1.4.2": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/9773353408\">bot CI job</a>): main:\nError running migrate-feedstock in container - error DuplicateKeyError raised:\n'while constructing a mapping',   in \"<unicode string>\", line 1, column 1:\n    build_platform: {osx_arm64: osx_64}\n    ^ (line: 1), 'found duplicate key \"build_platform\" with value \"{'linux_aarch64': 'linux_64', 'linux_ppc64le': 'linux_64'}\" (original value: \"{'osx_arm64': 'osx_64'}\")',   in \"<unicode string>\", line 3, column 1:\n    build_platform:\n    ^ (line: 3), '\n                    To suppress this check see:\n                        http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n                    ', '                    Duplicate keys will become an error in future releases, and are errors\n                    by default when using the new API.\n                    '",
  "1.4.3": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/10305384225\">bot CI job</a>): main:\nError running migrate-feedstock in container - error DuplicateKeyError raised:\n'while constructing a mapping',   in \"<unicode string>\", line 1, column 1:\n    build_platform: {osx_arm64: osx_64}\n    ^ (line: 1), 'found duplicate key \"build_platform\" with value \"{'linux_aarch64': 'linux_64', 'linux_ppc64le': 'linux_64'}\" (original value: \"{'osx_arm64': 'osx_64'}\")',   in \"<unicode string>\", line 3, column 1:\n    build_platform:\n    ^ (line: 3), '\n                    To suppress this check see:\n                        http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n                    ', '                    Duplicate keys will become an error in future releases, and are errors\n                    by default when using the new API.\n                    '",
  "1.4.4": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/10419861712\">bot CI job</a>): main:\nError running migrate-feedstock in container - error DuplicateKeyError raised:\n'while constructing a mapping',   in \"<unicode string>\", line 1, column 1:\n    build_platform: {osx_arm64: osx_64}\n    ^ (line: 1), 'found duplicate key \"build_platform\" with value \"{'linux_aarch64': 'linux_64', 'linux_ppc64le': 'linux_64'}\" (original value: \"{'osx_arm64': 'osx_64'}\")',   in \"<unicode string>\", line 3, column 1:\n    build_platform:\n    ^ (line: 3), '\n                    To suppress this check see:\n                        http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n                    ', '                    Duplicate keys will become an error in future releases, and are errors\n                    by default when using the new API.\n                    '"
 }
}